plugins/GSdx/GSDevice.h

```diff
@@ -52,6 +53,17 @@ enum ShaderConvert {
 	ShaderConvert_Count
 };
 
+enum ChannelFetch
+{
+	ChannelFetch_NONE  = 0,
```

Comment:
Thats camel case, snake case and uppercase all in one line. Enum classes are imo a better way to enable namespacing on enums than prefixes.